scripts/setupClearport.ts

```diff
@@ -0,0 +1,165 @@
+import { readFileSync, writeFileSync } from 'node:fs';
+
+import { ethers } from 'hardhat';
+import { type Signer, utils } from 'ethers';
+
+import type { TestERC20 } from '../typechain-types';
+
+const DEPLOYED_CONTRACTS_FILEPATH = './scripts/deployedContracts.json';
```

Comment:
let's make a config/ folder output values into `config/network.info.json`
use the config `config/name.config.json`

example:
`config/canarynet.config.json` > `config/canarynet.info.json`